site/components/Form/Checkbox/index.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { InputHTMLAttributes } from "react";
+import { cx } from "@emotion/css";
+
+import { Text } from "@klimadao/lib/components";
+
+import * as styles from "./styles";
+
+interface Props {
+  inputProps: InputHTMLAttributes<HTMLInputElement>;
+  label: string;
+  errorMessage?: string;
+}
+
+export const Checkbox = React.forwardRef<HTMLInputElement, Props>(
+  (props, ref) => {
+    const inputStyles = cx(
+      styles.baseStyles,
+      {
+        [styles.errorStyles]: !!props.errorMessage,
+      },
+      props.inputProps.className
+    );
+
+    return (
+      <div className={styles.checkboxContainer}>
+        <input
+          id={props.inputProps.id}
```

Comment:
is this prop required? what happens if you don't pass an id? I don't think typescript would enforce it as required the way this is right now